sailthru_import/test/test_sailthru_import.py

```diff
@@ -0,0 +1,15 @@
+from sailthru_import import sailthru_import
+from sailthru.sailthru_client import SailthruClient
+
+import unittest
+import mock
+from mock import patch
+
+
+@patch('sailthru.sailthru_client')
+class SailThruTestCase(unittest.TestCase):
+    def test_upload_csv_to_sailthru(self, mock_client):
+        mock_client.api_get.return_value.get_body.return_value = {'status': 'completed'}
```

Comment:
You can just create the client in this method, by calling `mock_client = Mock(spec=SailthruClient)`. That way, the mock will raise an error if any method is called on it that's not available on the actual client.